src/liballoc/vec_deque.rs

```diff
@@ -1624,18 +1624,18 @@ impl<T> VecDeque<T> {
         return elem;
     }
 
-    /// Splits the collection into two at the given index.
+    /// Splits the `VecDeque` into two at the given index.
     ///
-    /// Returns a newly allocated `Self`. `self` contains elements `[0, at)`,
-    /// and the returned `Self` contains elements `[at, len)`.
+    /// Returns a newly allocated `VecDeque`. `self` contains elements `[0, at)`,
+    /// and the returned `VecDeque` contains elements `[at, len)`.
     ///
     /// Note that the capacity of `self` does not change.
     ///
     /// Element at index 0 is the front of the queue.
     ///
     /// # Panics
     ///
-    /// Panics if `at > len`
+    /// Panics if `at` is out of bounds.
```

Comment:
This seems less descriptive than it was previously. `at` is a usize, so we know it cannot be negative, but it's not immediately obvious what the "bounds" would be.

Reply:
Nice catch, @shepmaster, especially since `at == len` is allowed. 
I wanted to get rid of the un-introduced value `len`, but in the end, it seems best to just keep it, so I have reverted the change.